File: 11044/11044.py
#!/usr/bin/python3
# @JUDGE_ID: mobluse 11044 Python 3 "Closed-form expression"

t = int(input())
for i in range(t):
    n, m = [int(x) for x in input().split()]
    print((n//3)*(m//3))

File: 11044/11044.pas
(* @JUDGE_ID: mobluse 11044 Pascal "Closed-form expression" *)
program UVa11044(input, output);

var t, n, m, i: integer;

begin
    readln(t);
    for i := 1 to t do
    begin
        readln(n, m);
        writeln((n div 3)*(m div 3));
    end
end.
